ru.yandex.Browser.yaml

```diff
@@ -0,0 +1,116 @@
+    config-opts:
+      - -Dbash_completion=false
+      - -Dman=false
+      - -Dvapi=false
+    cleanup:
+      - /etc
+      - /include
+      - ca.desrt.dconf.service
+      - dconf.service
+      - dconf-service
+      - '*.pc'
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/dconf/0.40/dconf-0.40.0.tar.xz
+        sha256: cf7f22a4c9200421d8d3325c5c1b8b93a36843650c9f95d6451e20f0bcb24533
+        x-checker-data:
+          type: gnome
+          name: dconf
+          stable-only: true
+      # From: https://src.fedoraproject.org/rpms/dconf
+      - type: patch
+        path: dconf-override.patch
+
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        tag: v2022.04
+        commit: 55a60d110301e8dd37b631503c3524ab7baaf7aa
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+
+  - name: yandex-browser
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/yandex_browser /app/tmp-install/data /app/tmp-codecs-install/data
+      - bsdtar -xf yandex-browser.rpm -C /app/tmp-install
+      - ar -x chromium-codecs-ffmpeg-extra.deb --output /app/tmp-codecs-install
+      - tar -xf /app/tmp-codecs-install/data.tar.xz -C /app/tmp-codecs-install/data
+      - mv /app/tmp-install/opt/yandex/browser/* /app/yandex_browser
+      - install -D -m755 /app/tmp-codecs-install/data/usr/lib/chromium-browser/libffmpeg.so /app/yandex_browser
+      - rm -rf /app/tmp-install /app/tmp-codecs-install yandex-browser.rpm chromium-codecs-ffmpeg-extra.deb
+      - install -D ru.yandex.Browser.desktop /app/share/applications/ru.yandex.Browser.desktop
+      - install -D ru.yandex.Browser.svg /app/share/icons/hicolor/scalable/apps/ru.yandex.Browser.svg
+      - install -D yandex-browser.sh /app/bin/yandex-browser
+      - install -D -t /app/etc cobalt.ini
+      - install -D -t /app/share/metainfo ru.yandex.Browser.metainfo.xml
+    sources:
+      - type: file
+        url: https://repo.yandex.ru/yandex-browser/rpm/stable/x86_64/yandex-browser-stable-23.1.5.751-1.x86_64.rpm
+        sha256: 5ef6f2becc6054e174457527a29005a5aadf30213c3198b21a52bb8e8fb1d639
+        dest-filename: yandex-browser.rpm
+        only-arches: [x86_64]
+        x-checker-data:
+          type: rpm-repo
+          package-name: yandex-browser-stable
+          root: https://repo.yandex.ru/yandex-browser/rpm/stable/x86_64
+      - type: file
+        url: https://launchpadlibrarian.net/637033261/chromium-codecs-ffmpeg-extra_108.0.5359.71-0ubuntu0.18.04.5_amd64.deb
+        sha256: 9cde2bbef1c5dff70621b1afb24676201ad824d6be9398796975db48782fab1e
+        dest-filename: chromium-codecs-ffmpeg-extra.deb
+        only-arches: [x86_64]
+      - type: file
+        path: cobalt.ini
+      - type: file
+        path: ru.yandex.Browser.metainfo.xml
+      - type: file
+        path: ru.yandex.Browser.svg
+      - type: file
+        path: ru.yandex.Browser.desktop
```

Comment:
It looks like the RPM already contains a.desktop and a .appdata file. Why did you not use those?

Reply:
RPM file contains different identifiers in appdata file, and different executable name in desktop file. Looks like it's less effort to just have ready to use files in repo than patch those coming from RPMs.

Reply:
no it's better to use the provided one. You can use rename commands to rename them. And `desktop-file-edit` to modify the desktop file.

Reply:
I've moved to provided .desktop file, but in conrary have no reason to move to upstream .appdata, especially as Flatpak version of it should contain Flathub-specific information.

For example: https://github.com/flathub/com.brave.Browser/blob/master/com.brave.Browser.metainfo.xml. There is a releases list, which doesn't apply to upstream appdata file.

Reply:
>  I've moved to provided .desktop file, but in conrary have no reason to move to upstream .appdata, especially as Flatpak version of it should contain Flathub-specific information.

This is untrue. There is nothing specific for flatpak, ie a flatpak compliant .desktop works everywhere. If need there is `desktop-file-edit` to tweak the upstream provided.

Reply:
I'm talking about appdata file, not desktop file. Desktop file is replaced with upstream one already.